resources/sass/app.scss

```diff
@@ -239,86 +237,86 @@ dl.import-meta .download-csv {
 }
 
 .button {
-    padding: $wikit-Button-medium-padding-vertical $wikit-Button-medium-padding-horizontal;
-    font-weight: $wikit-Button-font-weight;
-    line-height: $wikit-Button-line-height;
+    padding: $spacing-30 $spacing-75;
+    font-weight: $font-weight-bold;
```

Comment:
perhaps we could include the UI-text-M mixin here instead?

Reply:
done